addons/website_sale_coupon/tests/test_shop_sale_coupon.py

```diff
@@ -0,0 +1,162 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
```

Comment:
This file should be removed completely 🙂 
Thanks for taking the time to clean up the code 😄

Reply:
Ooopsie, seems like I missed something during the rebase :man_facepalming:
sorry for the ping :3